Networking/Networking/Model/WooPaymentsDepositsOverview.swift

```diff
@@ -315,6 +315,28 @@ public enum WooPaymentsDepositInterval: Equatable, GeneratedFakeable, GeneratedC
             return localizedDayName
         }
 
+        // Identifier for the weekday unit
+        // The weekday units are the numbers 1 through N (where for the Gregorian calendar N=7 and 1 is Sunday)
+        // https://developer.apple.com/documentation/foundation/calendar/component/weekday
+        public var dayToInt: Int {
```

Comment:
We already have a `dayIndex` variable here that I attempted to use initially, however the "current day" was off by one since is an index and the first day (Sunday) returned 0 rather than 1. This caused the date calculation to be incorrect when using Apple's weekday components, so I've added the new `dayToInt` variable to keep this PR limited.